packages/puppeteer-core/src/common/Input.ts

```diff
@@ -598,6 +598,33 @@ export class Mouse {
     }
   }
 
+  /**
+   * Resets the mouse to the default state. No buttons pressed; position at
```

Comment:
Those are implementation details. I'm not sure if we want to expose them.

Reply:
I think we should as they convey that the appropriate events will be send on the page side.

Reply:
I agree with Nik. Because we have also an option to reset without sending the page commands, so we need to clarify that the events will be dispatched. Perhaps: `Moves the mouse to 0,0 position and un-pressed all pressed buttons` or smth like that.